provider/azureProvider.js

```diff
@@ -102,27 +82,14 @@ return constants.providerName;
 return this.parsedBindings;
   }
 
-  getSetting (key) {
-    // Loop through environment variables looking for the keys, case insentivie
-    for (var k in process.env) {
-      if (process.env.hasOwnProperty(k)) {
-        if (k.toLowerCase() === key.toLowerCase()) {
-      return process.env[k];
-        }
-      }
-    }
-  }
+  Login() {
+    return login({ interactiveLoginHandler: (code) => {
+        this.serverless.cli.log(`Paste this code (copied to your clipboard) into the launched browser, and complete the authentication process: ${code}`);
```

Comment:
That implies users are going to go into the portal and explore their `Azure Active Directory` settings (is that really that common for new Azure users?) Can't we just keep the logging simple and expand it based on feedback? If we log super verbose messages, we run the risk of making it overwhelming/overcomplicated for everyone, as opposed to accommodating the few that would care to know a SP is being created.

I actually think the current logging for the `deploy` is already way too verbose as it is.